packages/grid/x-data-grid-premium/src/DataGridPremium/DataGridPremium.tsx

```diff
@@ -60,6 +60,16 @@ interface DataGridPremiumComponent {
   propTypes?: any;
 }
 
+/**
+ *
```

Comment:
First newline in these comments is superfluous, we can keep this more compact.

Reply:
And I think maybe there is a JSDoc annotation for these? Not sure, to be verified.

Reply:
There is one, but generally both work just fine in this case. Is it worth the effort of changing those? I cannot find any benefit from JSDoc annotation over MD style.